packages/styles/borders.js

```diff
@@ -0,0 +1,16 @@
+export const thin = {
+  borderWidth: '1px',
```

Comment:
All instances of `px` can actually be an integer, as integers are assumed to be using `px`.